helix-term/src/config.rs

```diff
@@ -43,11 +35,71 @@ impl Display for ConfigLoadError {
 }
 
 impl Config {
+    // avoids confusion between a default rust struct and default helix keys
+    /// Creates a new empty config
+    #[allow(clippy::new_without_default)]
+    pub fn new() -> Self {
+        Config {
+            theme: None,
+            unbind_default_keys: false,
+            keys: HashMap::new(),
+            editor: helix_view::editor::Config::default(),
+        }
+    }
```

Comment:
The original Default implementation still makes sense: it uses the default keys. I think this should be renamed as `Config::empty` or `Config::without_default_keys`

Reply:
I rewrote the feature to bring it up to the latest version of helix and no longer changed the config's `Default` implementation. The codebase has changed a bit so I don't think a `Config::empty` method is needed anymore